modules/tinymce/src/core/test/ts/browser/focus/HighlightOnFocusTest.ts

```diff
@@ -24,7 +24,7 @@ describe('browser.tinymce.core.focus.HighlightOnFocus', () => {
     assertIsHighlighted(editor);
   };
 
-  context('with highlight_on_focus: true', () => {
+  context('without highlight_on_focus option', () => {
     const hook = TinyHooks.bddSetup<Editor>({
       base_url: '/project/tinymce/js/tinymce',
       toolbar: 'forecolor',
```

Comment:
Should you remove the ` highlight_on_focus: true` just beneath?

Reply:
yes, done